### PR TITLE
fix(w3c/headers): don't add W3C to h2 for 'base' documents

### LINKED ISSUE #3542: How to remove "W3C Document"
Hi, I am using ReSpec to develop a profile of W3C DCAT for another international organisation:
 https://airservicesaustralia.github.io/dcat-profile/index.html

Is there a way to remove/replace the "W3C Document" text next to the date (under the main heading)?

### PR DESCRIPTION
**Pull request by marcoscaceres**
closes #3542

### CODE REVIEW

**Comment by sidvishnoi** [comment]
For base documents, it [now shows](https://respec-preview.netlify.app/?spec=https%3A%2F%2Fw3c.github.io%2Fgamepad%2F&version=https%3A%2F%2Fdeploy-preview-3545--respec-pr.netlify.app%2Frespec-w3c.js&specStatus=base) `Document {date}`, which looks a little odd. Is this the intended behavior?
image

File: tests/spec/w3c/headers-spec.js [hunk 1]
```diff
@@ -67,6 +67,16 @@ describe("W3C — Headers", () => {
         "has received the endorsement of the W3C and its Members. W3C recommends the wide deployment of this specification as a standard for the Web."
       );
     });
+
+    describe("specStatus - base", () => {
```
Comment (sidvishnoi) [inline tests/spec/w3c/headers-spec.js:71]: The previous describe block is `specStatus`. Should we merge this test in that block?

Comment (marcoscaceres) [reply tests/spec/w3c/headers-spec.js:71]: I thought of splitting them by type, as "base" is kinda special.

**Comment by marcoscaceres** [comment]
Yeah, good point... Just "Document date" is kinda weird.

**Review by sidvishnoi — APPROVED** [review]

**Pull request merged**